pkg/version/version.go

```diff
@@ -5,25 +5,26 @@ import (
 	"strings"
 
 	"github.com/tektoncd/cli/pkg/cli"
+	v1 "k8s.io/api/apps/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-const pipelineNamespace = "tekton-pipelines"
-
 // GetPipelineVersion Get pipeline version, functions imported from Dashboard
 func GetPipelineVersion(c *cli.Clients) (string, error) {
 	version := ""
+	namespaces := []string{"tekton-pipelines", "openshift-pipelines"}
 
 	listOptions := metav1.ListOptions{
 		LabelSelector: "app.kubernetes.io/component=controller,app.kubernetes.io/name=tekton-pipelines",
 	}
 
-	deployments, err := c.Kube.AppsV1().Deployments(pipelineNamespace).List(listOptions)
-	if err != nil {
-		return "", err
+	var deployments []v1.Deployment
+	for _, namespace := range namespaces {
+		d, _ := c.Kube.AppsV1().Deployments(namespace).List(listOptions)
```

Comment:
We should at least log the error :wink:

Reply:
it's a nit, not sure if it make sense really (to log the error)

Reply:
why would you list the deployments? i don't think that change much compared to the previous iteration of this patch, what about first trying the static ones and then listing deployment as last course ?

Reply:
PR updated as per your comment. Please let me know if any other change required.